src/Compilers/CSharp/Portable/Errors/ErrorCode.cs

```diff
@@ -1508,5 +1508,10 @@ internal enum ErrorCode
         ERR_RefReturnReadonlyNotField2 = 8411,
         ERR_ExplicitReservedAttr = 8412,
         ERR_TypeReserved = 8413,
+
+        //PROTOTYPE(ReadonlyRefs): make err IDs contiguous before merging to master. 
```

Comment:
>make err IDs contiguous before merging to master [](start = 35, length = 48)

There is no requirement to make number contiguous, but if we are leaving a gap, we should mark it with "Available" comment.